doc/ext/convert-svg-to-pdf.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+"""
+    Converts SVG images to PDF using chrome in case the builder does not
+    support SVG images natively (e.g. LaTeX).
+
+"""
+import subprocess
+
+from sphinx.errors import ExtensionError
+from sphinx.locale import __
+from sphinx.transforms.post_transforms.images import ImageConverter
+from sphinx.util import logging
+from errno import ENOENT, EPIPE, EINVAL
+import os
+import platform
+import tempfile
+
+if False:
+    # For type annotation
+    from typing import Any, Dict  # NOQA
+    from sphinx.application import Sphinx  # NOQA
```

Comment:
The `if` should be removed. Also, if you use `from __future__ import annotations` at the 1st line, you can just use `dict` instead of `Dict`.